catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswConstants.java

```diff
@@ -22,266 +22,699 @@
 
-    QName DC_IDENTIFIER_QNAME = new QName(DUBLIN_CORE_SCHEMA, "identifier");
+    public static final QName DC_IDENTIFIER_QNAME = new QName(DUBLIN_CORE_SCHEMA, "identifier");
 
-    QName DC_TITLE_QNAME = new QName(DUBLIN_CORE_SCHEMA, "title");
+    public static final QName DC_TITLE_QNAME = new QName(DUBLIN_CORE_SCHEMA, "title");
 
-    String CSW_OGCCORE_OUTPUT_SCHEMA = "OGCCORE";
+    public static final String CONSTRAINT_VERSION = "1.1.0";
 
-    String CONSTRAINT_VERSION = "1.1.0";
+    public static final String OGC_CSW_PACKAGE = "net.opengis.cat.csw.v_2_0_2";
 
-    String OGC_CSW_PACKAGE = "net.opengis.cat.csw.v_2_0_2";
+    public static final String OGC_FILTER_PACKAGE = "net.opengis.filter.v_1_1_0";
 
-    String OGC_FILTER_PACKAGE = "net.opengis.filter.v_1_1_0";
+    public static final String OGC_GML_PACKAGE = "net.opengis.gml.v_3_1_1";
 
-    String OGC_GML_PACKAGE = "net.opengis.gml.v_3_1_1";
+    public static final String OGC_OWS_PACKAGE = "net.opengis.ows.v_1_0_0";
 
-    String OGC_OWS_PACKAGE = "net.opengis.ows.v_1_0_0";
+    public static final String XML_SCHEMA_LANGUAGE = "http://www.w3.org/XML/Schema";
 
-    String XML_SCHEMA_LANGUAGE = "http://www.w3.org/XML/Schema";
+    public static final String CSW_NO_PREFIX_TITLE = "title";
 
-    String CSW_TYPE = "type";
+    public static final String CSW_NO_PREFIX_MODIFIED = "modified";
 
-    String CSW_TITLE = "title";
+    public static final String CSW_NO_PREFIX_CREATED = "created";
 
-    String CSW_MODIFIED = "modified";
+    public static final String GML_POINT = "Point";
 
-    String CSW_CREATED = "created";
+    public static final String GML_LINESTRING = "LineString";
 
-    String GML_POINT = "Point";
+    public static final String GML_POLYGON = "Polygon";
 
-    String GML_LINESTRING = "LineString";
+    public static final String BBOX_PROP = "ows:BoundingBox";
 
-    String GML_POLYGON = "Polygon";
+    public static final String SPATIAL_PROP = "dct:Spatial";
 
-    String BBOX_PROP = "ows:BoundingBox";
+    public static final String METERS = "METERS";
 
-    String SPATIAL_PROP = "dct:Spatial";
+    public static final String CRS = "crs";
 
-    String METERS = "METERS";
+    public static final String SRS_NAME = "EPSG:4326";
 
-    String CRS = "crs";
+    public static final String SRS_URL = "urn:x-ogc:def:crs:EPSG:6.11:4326";
 
-    String SRS_NAME = "EPSG:4326";
+    public static final String OWS_UPPER_CORNER = "UpperCorner";
 
-    String SRS_URL = "urn:x-ogc:def:crs:EPSG:6.11:4326";
+    public static final String OWS_LOWER_CORNER = "LowerCorner";
 
-    String OWS_UPPER_CORNER = "UpperCorner";
+    public static final double DEGREES_TO_RADIANS = Math.PI / 180.0;
 
-    String OWS_LOWER_CORNER = "LowerCorner";
+    public static final double RADIANS_TO_DEGREES = 1 / DEGREES_TO_RADIANS;
 
-    double DEGREES_TO_RADIANS = Math.PI / 180.0;
+    public static final double EARTH_MEAN_RADIUS_METERS = 6371008.7714;
```

Comment:
I wondering if you should drop in a javadoc that indicates the official source of this value.